server/plugin.go

```diff
@@ -41,6 +41,19 @@ const (
 
 	metricsExposePort          = ":9094"
 	updateMetricsTaskFrequency = 15 * time.Minute
+
+	discardedReasonUnableToGetMMData         = "unable_to_get_mm_data"
```

Comment:
I think there are some constants with the same prefix defined in some other file.

Reply:
that was related to Teams data https://github.com/mattermost/mattermost-plugin-msteams-sync/blob/f34054a67779e0885fa9febe4e94c86914759e5b/server/handlers/handlers.go#L37